core/ir/instr_api.h

```diff
@@ -722,6 +724,11 @@ DR_API
 int
 instr_get_opcode(instr_t *instr);
 
+DR_API
+/** Returns \p instr's set of categories (set of DR_INSTR_CATEGORY_ constants). */
```

Comment:
Always make references to other types actual links in Doxygen comments: `#dr_instr_category_type_t`